marshalItem.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+var _ = require('lodash/fp');
```

Comment:
Yeah, I had been using it but I felt that it actually made the code less clear so I left it as it was.